common/source/docs/common-FPort-receivers.rst

```diff
@@ -0,0 +1,111 @@
+.. _common-FPort-receivers:
+
+===============
+FPort Receivers
+===============
+
+FPort protocol combines SBus RC control information being sent to the autopilot with bi-directional telemetry information to/from the autopilot on a single wire high baud rate bus.
+
+Many FRSky X series and R series receivers have this capability, either natively or with firmware upgrade.
+
+Connection to the autopilot and configuration varies from autopilot to autopilot since their UART capabilities vary. Most F4 cpu based autopilots do not controllable inverters in front of the UART TX and RX, and require either an external bi-directional inverter, as SPort connections do (see :ref:`frsky_cables` ), or the use of the "uninverted FPort pad/pin" some receivers provide since the FPort protocol has opposite signaling levels than normal UART operation.
```

Comment:
"do not controllable" should be "do not have controllable"

Reply:
fixed

Reply:
fixed